xbox360_controller.py

```diff
@@ -0,0 +1,297 @@
+#  Copyright (c) 2017 Jon Cooper
+#
+#  This file is part of pygame-xbox360controller.
+#  Documentation, related files, and licensing can be found at

```

Comment:
For any borrowed code, we need to check on licensing.     We will likely want to release our code under some kind of license, and it must be compatible with the various licenses of all of the borrowed code.